client/src/components/v3/status/CameraStatus.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Col, Table } from 'react-bootstrap';
+import OnlineStatusPill from 'components/common/OnlineStatusPill';
+
+export interface CameraStatusProps {
+  cameraName: string;
+  online: boolean;
+  ip: string | null;
+  battery: number | null;
+  videoFeedStatus: boolean | null;
+}
+
+/**
+ * Status for Cameras
+ *
+ * @property props Props
+ * @returns component
+ */
+export default function CameraStatus({
+  cameraName,
+  online,
+  ip,
+  battery,
+  videoFeedStatus,
+}: CameraStatusProps) {
+  return (
+    <Col md xl="12" className="my-2">
+      <span>
+        <b>{cameraName}</b> <OnlineStatusPill isOnline={online} />
+      </span>
+
+      {/* Only show more information if the camera is online */}
+      {online && (
+        <>
+          {/* IP address */}
+          <p style={{ fontSize: '0.75rem', color: 'gray' }}>{ip}</p>
+
+          <Table bordered hover>
+            <tbody>
+              {/* Battery Voltage */}
+              <tr>
+                <td>
+                  <strong>Battery</strong>
+                </td>
+                <td>{`${battery ?? '-'}  %`}</td>
+              </tr>
+
+              {/* Video Feed Status */}
+              <tr>
+                <td>
+                  <strong>Video Feed</strong>
+                </td>
+                <td>{videoFeedStatus ? 'RECORDING' : 'OFFLINE'}</td>
```

Comment:
Can you confirm whether this is meant to represent either
- Video feed is displayed, or
- Video feed is being recorded?

I'd rename the variable to `videoFeedEnabled` or `videoFeedRecording` respectively to avoid confusion.

Reply:
I think it's referring to `videoFeedEnabled` but unsure. I've set it to use the `useVideoFeedStatus` hook that represents whether the video feed is displayed. @Blake-Haydon ?

Reply:
Changed to `videoFeedEnabled`
bf1473d